requirements.txt

```diff
@@ -19,7 +19,7 @@ django==1.7.4
 django_extensions==1.5
```

Comment:
If all dependencies are now in `setup_requirements.py`, I would remove this file (and possibly move comments in there)